README.md

```diff
@@ -60,16 +72,30 @@ Arguments:
 - `size_display` tuple (int, int): width and height of the canvas (minimum 400)
 - `size_world` float: extend on the 3D scene in all directions
 
+Important attributes/mdethods:
+- `self.editor` : display an editor of the graphical parameters
```

Comment:
Why "self" and not "lw"?

Reply:
I change this to be similar to the help(LsystemWidget). Now you have editor and get_lstring(self).